src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--28">
+      <div class="calendar__day-1 start-day_sat"></div>
```

Comment:
- start-day should be a modifier for the calendar block
- don’t need to add 1,2,3 to each day (use nth-child)

```suggestion
    <div class="calendar calendar--month-length_28 calendar--start-day_sat">
      <div class="calendar__day"></div>
```